content/chainguard/chainguard-images/working-with-images/scanners/grype-tutorial/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Using the grype Container Scanner"
```

Comment:
I like it. I'm going to leave grype lower case since they're pretty into their lower case stuff and consistent about it. Admittedly kind of painful to write when starting sentences. Final title:

Using grype to Scan Container Images for Vulnerabilities